include/dlaf/eigensolver/band_to_tridiag/mc.h

```diff
@@ -535,4 +798,548 @@ TridiagResult<T, Device::CPU> BandToTridiag<Backend::MC, D, T>::call_L(
   return {std::move(mat_trid), std::move(mat_v)};
 }
 
+std::ostream& operator<<(std::ostream& out, matrix::SubTileSpec spec) {
+  out << spec.origin << "+" << spec.size;
+  return out;
+}
```

Comment:
This should presumably be removed before merging.

Reply:
Yes, it is a remainder of debug printing...